src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObj = {};
+  const clearProperties = sourceString
+    .replaceAll('\n', '')
+    .split(';')
+    .map(
+      rule => rule.split(':'))
+    .filter(
+      rule => rule.length === 2);
```

Comment:
```suggestion
  const clearProperties = sourceString
    .replaceAll('\n', '')
    .split(';')
    .map(rule => rule.split(':'))
    .filter(rule => rule.length === 2);
```

Reply:
don't understand why you need to filter the array by `length === 2`? mayble let's try `length > 0`?

Reply:
I used it to filter arrays with empty strings/strings with spaces. 
.filter(([key, value]) => key != false); also can be used for that, but I didn't want to use this equality operator because of linter. 

Please give some advice for better solving.